config/earlybird.json

```diff
@@ -57,6 +57,6 @@
   "display_threshold_level": 3,
   "display_confidence_threshold_level": 2,
   "earlybird_config_url": "https://raw.githubusercontent.com/americanexpress/earlybird/main/config/earlybird.json",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
No need to bump this version. Github actions take care of it -> https://github.com/americanexpress/earlybird/blob/main/.github/workflows/release.yml#L28